### DISCUSSION TITLE
Standalone `x-data` causing `x is not defined` error

### DISCUSSION THREAD

**Discussion by jacksleight** [alpinejs/alpine#2659] [1. Help]
I'm implementing the Tooltip component from Alpine Components, and in the example code the `x-data` attribute is defined as just `x-data`, with no equals sign or quotes. Which makes sense, no value is needed.

But when I try this I get an error in the console:

> ReferenceError: x is not defined

Replacing `x-data` with `x-data=""` fixes it.

Any idea what the problem is?

**Comment by danddanddand** [comment]
I use x-data on it's own all the time and never saw this. Can you provide codepen or something?

**Reply by jacksleight** [reply]
Never mind, I'm an idiot 😄 .

I was adding `x-data` to a Laravel Blade component:

```blade
<x-button x-data x-tooltip="Hello World!">Example</x-button>
```

Blade treats the `x-data` as a boolean attribute and turns it into `x-data="x-data"` in the output HTML, causing the error.
